pkg/authorization/api/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package api is the internal version of the API.
+// +genconversion=true
```

Comment:
OK this is not good:

https://gist.github.com/mfojtik/4208f4986bac5f3d0b9c6cc08480aae2#file-gistfile1-txt-L64

@smarterclayton @deads2k any idea?